packages/react/__mocks__/@fluentui/utilities.ts

```diff
@@ -8,14 +8,14 @@ declare function setTimeout(cb: Function, delay: number): number;
 // https://github.com/facebook/jest/issues/3465
 // Mock impl inspired from issue.
 class MockAsync extends Async {
-  private _timeoutId: number | null;
+  private _timeoutId: number | undefined;
```

Comment:
- its because how Typescript defines its standard library 

```
clearInterval(handle?: number): void;
```

handle is optional (`undefined` ) OR `number`